data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -36,6 +73,54 @@ def test_init(self, mock_etl, mock_paths):
         assert etl.GEOID_FIELD_NAME == "GEOID10"
         assert etl.GEOID_TRACT_FIELD_NAME == "GEOID10_TRACT"
 
+    @mock.patch("data_pipeline.utils.requests")
+    def test_extract(self, requests_mock, mock_paths):
```

Comment:
Yeaaa good catch! To be honest I am much more familiar with this individualized way of doing a `@mock.patch` on the method, but it is out-of-sync with how the rest of the test fixtures work and I shouldl take the extra step to learn something new today and bring more consistency to the codebase. 

Thanks for flagging this, I will refactor!

Reply:
@lucasmbrown-usds After reflecting on this further, I realized that if we're abstracting these tests to a base test class (`TestETLBase`) then there probably won't be much added benefit of promoting this segment of code to a pytest fixture since other ETL classes would simply use `super().test_extract()` instead of the `requests_mock` directly. I'm all for learning new stuff, haha, but it seems like you could de-scope this from the immediate refactor tasks you outlined below